docsrc/imap/concepts/deployment/databases.rst

```diff
@@ -213,13 +213,19 @@ File format not selectable.
 
 .. _imap-concepts-deployment-db-search:
 
-Search Indexes (cyrus.squat, <userid>.xapianactive)
+Search Indexes (cyrus.squat, <userid>.xapianactive, cyrus.indexed.db)
 ---------------------------------------------------
 
-This is either cyrus.squat in each folder, or if you're using xapian a single
-<userid>.xapianactive file listing active databases by tier name and number.
+This is either cyrus.squat in each folder, or if you're using Xapian a single
+<userid>.xapianactive file listing active databases with tier name and number.
 
-File type can be: `twoskip`_ (default), `flat`_, or `skiplist`_.
+cyrus.indexed.db is used by the Sphinx and Xapian search engines.  Its file type
```

Comment:
The Sphinx engine doesn't exist on master anymore, it was removed a while ago cause it was unmaintained and broken.  (It's still on 3.0, but is also unmaintained and probably broken -- e.g. #2748)